lib/Zend/Locale/Data.php

```diff
@@ -985,7 +1001,9 @@ public static function getContent($locale, $path, $value = false)
         $val = urlencode($val);
         $id  = self::_filterCacheId('Zend_LocaleC_' . $locale . '_' . $path . '_' . $val);
         if (!self::$_cacheDisabled && ($result = self::$_cache->load($id))) {
```

Comment:
btw if I understand correctly lack of 
```
        // add runtime cache to avoid callng cache backend multiple times during one request
        if (isset(self::$_localCache[$id])) {
            return self::$_localCache[$id];
        }
``` 
before the `if` will trigger `$_cache->load($id)` and data is not taken from runtime

Noticed that after the patch we anyway have many `Zend_LocaleC_` coming from cache

Reply:
can you propose a patch/pull request? thanks!

Reply:
https://github.com/OpenMage/magento-lts/pull/1018
I've pushed the change, and tested it.